types/react-elm-components/package.json

```diff
@@ -12,9 +12,10 @@
         "@types/react-elm-components": "workspace:."
     },
     "owners": [
+    {
```

Comment:
```suggestion
```

Reply:
Oh, shoot, didn't see that one. Thank you for fixing it for me!